data/human/human missions.txt

```diff
@@ -10297,3 +10298,1578 @@ mission "Patch Stone Waiting"
+			choice
+				`	"I'll come back. Don't worry."`
+				`	"We'll see. No promises."`
+				`	"I do what I want."`
+			`	Tavish nods, "In that case we've quite a bit of work to do, Captain. Take care."`
+			`	He starts working on prepping the dig site, while Kormak escorts you back to your ship.`
+			`	Kormak takes your hand in his. "May the stones guide you," he says, and then disappears back into the jungle.`
+				decline
+
+			label thepug
+			`	Tavish perks up, "Them again? How interesting. I wonder how long they have been meddling in the affairs of men."`
+			`	"Maybe not lies." Kormak admits, and then looks down as if in deep thought.`
+			`	"But maybe not whole story," he says finally.`
+			`	The large man shakes his head and frowns. "Friend." He says the word slowly, carefully enunciating each syllable.`
+			`	"If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf, know that is my home."`
+				goto pugout
+
+	on visit
+		dialog `You land on <planet>, but realize that Kormak and Tavish and their equipment are on one of your escorts who hasn't entered the system yet. Better depart and wait for them.`
+
+event "Tavish and Kormak Expedition"
+event "Tavish and Kormak Late"
+event "Tavish and Kormak Abandoned"
+
+
+mission "Tavish and Kormak 7"
+	landing
+	name "Alexandria awaits"
+	description "Transport Tavish and Kormak and their cargo to <planet>."
+	to offer
+		has "event: Tavish and Kormak Expedition"
+		not "event: Tavish and Kormak Abandoned"
+		not "event: Tavish and Kormak Late"
+	source "New India"
+	destination "Alexandria"
+	passengers 2
+	cargo "Scientific Research Equipment" 90
+	blocked "You do not have enough space to pick up Tavish and Kormak with their tons of cargo. Return here when you have 90 tons free."
+	on offer
+		conversation
+			`As you enter New India's atmosphere you pick up a faint signal from the Ashoka continent. It's a transport request from Tavish and Kormak.`
+			choice
+				`	(Land on Ashoka to pick up Tavish and Kormak.)`
+					goto Ashoka
+				`	(Ignore the signal.)`
+			`	You ignore the signal and land in the regular New India spaceport.`
+				defer
+
+			label Ashoka
+			`	Tavish and Kormak are waiting with all their equipment near the stony outcrop where you landed before. They seem more than a little relieved to see you.`
+			`	"Knew we could count on you," Tavish happily says as he oversees the loading up of the equipment.`
+			`	"Honorable," Kormak adds.`
+			`	Tavish gives you a serious look. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on the Alexandria station in the Deep. Will you do us this last favor?"`
```

Comment:
```suggestion
			`	Tavish gives you a serious look. "Captain <last>, old chap, we need you to take us to the esteemed headquarters of the Society of Gentlemanly Exemplary Exploration on Alexandria Station in the Deep. Will you do us this last favor?"`
```
The station is actually "Alexandria Station" iirc, it's just that "Alexandria Station" was too long to display on the map so MZ left it as "Alexandria". I think it's the same story with Mordente-Bridi, which is officially "Mordente-Bridi Station".